src/cryptoadvance/specter/templates/includes/merkletooltip.html

```diff
@@ -0,0 +1,9 @@
+<i class="tool-tip__icon">i</i>
+<p class="tool-tip__info">
+  <span class="info">
+      <span class="info__title">Merkle Proof Validation<br><br></span>
+      Specter-Desktop validates <a href="https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki">BIP 37</a> Merkle Proofs from your full node <strong>guaranteeing</strong> that transactions displayed as confirmed have been included in the specified block hash.<br><br>
+      However, a compromised full node could return a fake block that was never mined into bitcoin's blockchain!
```

Comment:
```suggestion
      However, a compromised full node could return a fake block that was never mined into Bitcoin's blockchain!
```
Capitalizes `Bitcoin` as we are talking about the protocol.